recipe/meta.yaml

```diff
@@ -12,19 +12,19 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 3
-  skip: True  # [osx or linux or py3k]
+  number: 1
+  skip: True  # [osx or linux or py27]
   features:
-    - vc14
-  msvc_compiler: 14.0
+    - vc14  # [py35 or py36]
```

Comment:
Could you please revert these few lines? The selector here doesn't make much sense and we need `msvc_compiler`.